src/main/java/Lesson5/Task3/Main.java

```diff
@@ -0,0 +1,19 @@
+package Lesson5.Task3;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+public class Main {
+    public static void main(String[] args) {
+        ArrayList <Integer> list = new ArrayList<>();
+        list.add(5);
+        list.add(2);
+        list.add(9);
+        list.add(4);
+        list.add(7);
+        Iterator<Integer> iterator = list.iterator();
+        while (iterator.hasNext()){
+            System.out.println(iterator.next()+1);
```

Comment:
Here you just print the +1 value, but we need to do +1 for list, we need to save new values in list